tox.ini

```
@@ -310,6 +316,7 @@ deps =
     gevent12: gevent>=1.2,<1.3
     gevent13: gevent>=1.3,<1.4
     gevent14: gevent>=1.4,<1.5
+    grpc: grpcio
```

```suggestion
    grpclatest: grpcio
```
I think this is the convention we're following for the other libraries